mdk/src/main/resources/META-INF/mods.toml

```diff
@@ -59,4 +67,4 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed mollis lacinia magn
 # This version range declares a minimum of the current minecraft version up to but not including the next major version
     versionRange="[@MC_VERSION@,@MC_NEXT_VERSION@)"
     ordering="NONE"
-    side="BOTH"
+    side="BOTH"
```

Comment:
Your IDE is removing line endings again :/

Reply:
i cherry pick from 1.18.2

Reply:
which is a cherry pick from 1.19.2